aptly/aptly_config.sls

```diff
@@ -22,33 +33,33 @@ aptly_rootdir:
 
 aptly_conf:
   file.managed:
-    - name: {{ salt['pillar.get']('aptly:homedir', '/var/lib/aptly') }}/.aptly.conf
-    - source: salt://aptly/files/.aptly.conf.jinja
+    - name: {{ aptly.homedir }}/.aptly.conf
+    - source: salt://aptly/files/aptly.conf.jinja
```

Comment:
If it is a 'jinja' template, it should go in /aptly/templates.
This is more of a PEP8 style guide kind of thing - eg.. nothing over 80 columns :)